Vcom/Functions/fn_HearingAids.sqf

```diff
@@ -5,12 +5,8 @@ if (_weapon isEqualTo "Put" || {_weapon isEqualTo "Throw"}) exitwith {};
 if ((group _unit) getVariable ["VCM_NOFLANK",false]) exitWith {};
 
 //Check if unit has suppressor on weapon.
-private _ItemList = weaponsitems _unit;
-private _Return = true;
-
-private _Atch = ((_ItemList select 0) select 1);
-
-if (_Atch isEqualTo "" && {!(_Atch in VCM_ATTACHMENTIGNORE)}) then {_Return = false;};
+private _Atch = player weaponAccessories currentMuzzle player param [0, ""];
+private _Return = (!(_Atch isEqualTo "")) && {getNumber(configFile >> "CfgWeapons" >> _Atch >> "ItemInfo" >> "AmmoCoef" >> "audibleFire") < 1};
```

Comment:
With `VCM_ATTACHMENTIGNORE` no longer needed, can you remove it from the configs in this pull?

https://github.com/genesis92x/VcomAI-3.0/blob/dfac639af4f4546e9a181d7c4cee224445aec494/Vcom/Functions/VcomAI_DefaultSettings.sqf#L41

https://github.com/genesis92x/VcomAI-3.0/blob/dfac639af4f4546e9a181d7c4cee224445aec494/userconfig/VCOM_AI/AISettingsV3.hpp#L41